src/main/java/net/masterthought/cucumber/json/support/Status.java

```diff
@@ -38,8 +38,15 @@ public String getLabel() {
         return String.valueOf(name().charAt(0)).toUpperCase() + name().substring(1).toLowerCase();
     }
 
-    public String toHtmlClass() {
+    public String toHtmlClass(String name) {
         // TODO: should be moved out of this class
-        return "<div class=\"" + this.getName().toLowerCase() + "\">";
+        String uniqueIdentifier = name.trim().toLowerCase().replaceAll(" ", "");
+        return "<div class=\"" + this.getName().toLowerCase() + "\" cuc=\""+uniqueIdentifier+"\">";
     }
-}
\ No newline at end of file
+
+    public String toHtmlClassScen(String name) {
+        // TODO: should be moved out of this class
+        String uniqueIdentifier = name.trim().toLowerCase().replaceAll(" ", "");
```

Comment:
i think its no longer needed, since i set name parameter to randomUUID, so there is no need to remove spaces etc. i will try to remove it.